### PR TITLE
feature: server

### PR DESCRIPTION
**Pull request by Nukbutcu**

### CODE REVIEW

**Review by konrascher3 — APPROVED** [review]
Sieht gut aus

File: package.json [hunk 1]
```diff
@@ -2,17 +2,24 @@
   "name": "react-bare-bone-template",
   "version": "0.1.0",
   "private": true,
+  "type": "module",
   "scripts": {
     "build": "react-scripts build",
-    "spj": "npx sort-package-json@1.48 package.json",
     "dev": "react-scripts start",
-    "stylelint": "stylelint 'src/*.{js,jsx}'"
+    "dev-server": "nodemon server.js localhost 1337",
```
Comment (Git-Hannah) [inline package.json:9]: does the localhost info go here, too?? I thought only in the server.js! ?

**Review by Git-Hannah — APPROVED** [review]
Just had a question, but looks good to lgtm!

File: package.json (hunk 1 above)
Comment (Nukbutcu) [reply package.json:9]: I believe that will be autoamtic through the const port. But I am not sure

Comment (Git-Hannah) [reply package.json:9]: that is what I thought! so to write it here is redundant? it doesnt break the code in any way?

Comment (Nukbutcu) [reply package.json:9]: i think so, but no guarantee 😁

**Pull request merged**